libconcurrent/primitives/primitives.c

```diff
@@ -135,10 +135,12 @@ inline void freeMemory(void *ptr, size_t size) {
 }
 
 inline int64_t getTimeMillis(void) {
-    struct timeb tm;
+    struct timespec tm;
 
-    ftime(&tm);
-    return 1000 * tm.time + tm.millitm;
+    if (clock_gettime(CLOCK_MONOTONIC, &tm) == -1) {
```

Comment:
There are some issues in WSL:
https://github.com/Microsoft/WSL/issues/3840
but i can live with that.

Reply:
I tested it in WSL2 and it seems to work correctly. Dis you test it with WSL1?

Reply:
I tested in OpenSUSE 15.2 over windows 20H2 + wsl1 and it works fine. Thus, I mark this point as resolved.